### DISCUSSION TITLE
Questions regarding @react-pdf/renderer

### DISCUSSION THREAD

**Discussion by mohnish-saini** [diegomura/react-pdf#1344] [Q&A]
Hi 

I am using @react-pdf/renderer package for my enterprise application. It is used for creating the pdf. Following is the one issue I am getting:

1. **Facing issues in rendering large content**. I am not facing any problem with rendering smaller content. On UI, I am having certain free text boxes. If I enter smaller content (few lines/words) then its not an issue.

But when I enter few lines in those text boxes, there is a problem and browser Chrome or Edge both becomes not responding.

this line of code is breaking and not able to create blob:

**const blob = await asPdf.toBlob();**

Please let me know how to fix this issue. It's very urgent from my end and I request you to provide assistance. 

Thanks
Mohnish Saini

**Comment by diegomura** [comment]
Hey! This sometimes happens because a page cannot be broken in several pages. Do you have a `wrap={false}` content somewhere that might be bigger than the page height? Otherwise, I'll need a way to replicate it to figure out what's going on

**Reply by mohnish-saini** [reply]
Hi, Thanks for your response. And I am using wrap={true}.

My pdf contains one section of text only. If text length become more than certain characters (alphanumeric, spaces only) then it breaks. Number of characters are not fixed. Sometimes it break at 1670 characters, sometime at 1800 or even more.

Following is the code snippet:

const generatePDF = async () => (
    
      const doc = <DocumentPdf />;
      const asPdf = pdf([]);
      asPdf.updateContainer(doc);
      **const blob = await asPdf.toBlob();**
      saveAs(blob, 'document.pdf');
    }
 
For larger text, above line of code (in bold) breaks down, not able to create the blob and browser becomes not responding. Eventually, browser has to be killed.

Please assist on this issue. Thanks
